src/components/calculator.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+
+class Calculator extends Component {
+  constructor() {
+    super();
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="calculator">
+        <div className="result">
+          <input type="text" />
+        </div>
+        <div className="row-flex">
+          <button className="btn" type="submit">AC</button>
+          <button className="btn" type="submit">+/-</button>
+          <button className="btn" type="submit">&#247;</button>
+          <button className="btn" type="submit">&#43;</button>
+        </div>
+        <div className="row-flex">
+          <button className="btn" type="submit">7</button>
+          <button className="btn" type="submit">8</button>
+          <button className="btn" type="submit">9</button>
+          <button className="btn" type="submit">&#215;</button>
+        </div>
+        <div className="row-flex">
+          <button className="btn" type="submit">4</button>
+          <button className="btn" type="submit">5</button>
+          <button className="btn" type="submit">6</button>
+          <button className="btn" type="submit">&#8722;</button>
+        </div>
+        <div className="row-flex">
+          <button className="btn" type="submit">1</button>
+          <button className="btn" type="submit">2</button>
+          <button className="btn" type="submit">3</button>
+          <button className="btn" type="submit">&#215;</button>
```

Comment:
- [ ] Please make sure to use the correct symbol for each place and avoid duplication. Currently, you have no `%` symbol. Also, there is two `multiplication` sign. Please fix it.

### Your design

![Screenshot from 2021-08-10 20-45-07](https://user-images.githubusercontent.com/22434660/128893424-603510b7-3738-4a54-bae3-f531a98f056c.png)

### Given design

![Screenshot from 2021-08-10 20-48-42](https://user-images.githubusercontent.com/22434660/128893622-b08fc46e-f651-4e05-a86e-7d4bb2c5cefc.png)